### DISCUSSION TITLE
Named internal links in revealjs presentation?

### DISCUSSION THREAD

**Discussion by hmu-stats** [quarto-dev/quarto-cli#9935] [Q&A] [answered]
### Description

My goal is to hyperlink to another slide in my revealjs slide deck. One way would be to link to the specific slide number (e.g. `#/2/5`), however, this is obviously fragile, as I add further slides. Is it somehow possible to add a named #ID anchor to a slide and jump to this anchor?

**Comment by hmu-stats** [comment, marked as answer]
Ah, I realize it is possible with 
```

## Source slide
[Link to target slide](#id-name)

## Target slide {#id-name}
```

It initially wasn't working in my case, because i had custom styling in the target slide title and tried it like this:
```
## [Target slide]{style="font-size: 35px" #id-name}
```
This doesn't work, but the following does:
```
## [Target slide]{style="font-size: 35px"}{#id-name}
```

**Reply by cscheid** [reply]
Attribute declarations in Pandoc need to happen in a specific order - identifier, classes, and then attributes. So `{#id-name style="font-size: 35px"}` should work.

**Comment by sylvaticus** [comment]
And how to add a "back" button to the destination slide without manually referring to the origin slide ?

**Reply by mcanouil** [reply]
Please open a new discussion as your question is a new one.

And this thread is more than 2 years old.
